### PR TITLE
Fix/fold wp cli into hookdocs site

### LINKED ISSUE #281: Fold WP-CLI docs into hookdocs site
**Is your enhancement related to a problem? Please describe.**
In order to continue evolving our [hookdoc site](https://10up.github.io/classifai/) into a more full-featured developer documentation site, we should look to port our [WP-CLI commands docs](https://github.com/10up/classifai/#wp-cli-commands) into the hookdoc site and rename it to ClassifAI Developer Documentation ([like what ElasticPress does](https://10up.github.io/ElasticPress/)).  Similarly adding a simple reference in the README.md to our developer docs to ensure they're clearly visible and available for folks looking for that level of info.

**Describe the solution you'd like**

**Designs**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by ActuallyConnor**
### Description of the Change

- Port our WP-CLI commands docs into the hookdoc site
- Rename it to ClassifAI Developer Documentation (like what ElasticPress does). 
- Add a simple reference in the README.md

[https://github.com/10up/classifai/issues/281](https://github.com/10up/classifai/issues/281)

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues

[https://github.com/10up/classifai/issues/281](https://github.com/10up/classifai/issues/281)

### Changelog Entry

Port WP-CLI commands docs into the hookdoc site. Rename the site to ClassifAI Developer Documentation. Add a simple reference to the hookdoc site in the README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,149 +124,7 @@ Note that [Computer Vision](https://docs.microsoft.com/en-us/azure/cognitive-ser
...
-default: `100`.
-
-**`--skip=<skip>`**
-
-Skip first N images.
-
-default: `false`.
+- Check out the [ClassifAI docs](https://10up.github.io/ClassifAI/).
 
 ### ClassifAI Settings Commands
```
Comment (faisal-alvi) [inline README.md]: Please also remove the "**ClassifAI Settings Commands**" section as it is also covered in the docs. The next title after the "Check out.." line should be "Frequently Asked Questions".

File: README.md [hunk 2]
```diff
@@ -17,6 +17,8 @@
 * [Changelog](#changelog)
 * [Contributing](#contributing)
 
+- Check out the [ClassifAI docs](https://10up.github.io/ClassifAI/).
+
```
Comment (faisal-alvi) [inline README.md]: This line is not required. It's already added below. Please remove from here.

File: README.md [hunk 3]
```diff
@@ -17,6 +17,8 @@
 * [Changelog](#changelog)
 * [Contributing](#contributing)
 
+- Check out the [ClassifAI docs](https://10up.github.io/ClassifAI/).
```
Comment (faisal-alvi) [inline README.md]: The link path is case-sensitive. Please change `ClassifAI` to `classifai`

File: README.md [hunk 4]
```diff
@@ -122,149 +124,7 @@ Note that [Computer Vision](https://docs.microsoft.com/en-us/azure/cognitive-ser
...
-Limit number of images to crop.
-
-default: `100`.
-
-**`--skip=<skip>`**
-
-Skip first N images.
-
-default: `false`.
+- Check out the [ClassifAI docs](https://10up.github.io/ClassifAI/).
```
Comment (faisal-alvi) [inline README.md]: The link path is case-sensitive. Please change `ClassifAI` to `classifai`

**Review by faisal-alvi — CHANGES_REQUESTED** [review]

**Comment by ActuallyConnor** [comment]
@faisal-alvi all the requested changes should be made now 😃

**Review by faisal-alvi — APPROVED** [review]

**Comment by ActuallyConnor** [comment]
@faisal-alvi @jeffpaul any possibility we could add the Hacktoberfest tag to the issue?

**Comment by jeffpaul** [comment]
@ActuallyConnor sure, let me do that, though not sure if Digital Ocean will count it after the fact but worth a shot!

**Comment by ActuallyConnor** [comment]
It worked 👍

**Pull request merged**